tendermint/src/rpc/event_listener.rs

```diff
@@ -171,65 +154,36 @@ impl response::Response for TxResultResult {}
 
 /// Tendermint ABCI Events
 #[derive(Serialize, Deserialize, Debug, Clone)]
-struct TmEvent {
+pub struct TmEvent {
     #[serde(rename = "type")]
     event_type: String,
     attributes: Vec<Attribute>,
 }
 /// Event Attributes
 #[derive(Serialize, Deserialize, Debug, Clone)]
-struct Attribute {
+pub struct Attribute {
     key: String,
     value: String,
 }
 
-/// Block Results
-#[derive(Serialize, Deserialize, Debug, Clone)]
-pub struct RPCBlockResult {
-    query: String,
-    data: BlockResultData,
-    events: HashMap<String, Vec<String>>,
-}
-impl response::Response for RPCBlockResult {}
-
-/// Block Results data
-#[derive(Serialize, Deserialize, Debug, Clone)]
-struct BlockResultData {
-    #[serde(rename = "type")]
-    data_type: String,
-    value: BlockValue,
-}
-
 ///Block Value
 #[derive(Serialize, Deserialize, Debug, Clone)]
-struct BlockValue {
+pub struct EventDataNewBlock {
     block: Option<Block>,
+
+    // TODO these should be the same as abci::responses::BeginBlock
```

Comment:
```suggestion
    // TODO(liamsi): these should be the same as abci::responses::BeginBlock
```

Maybe worth an issue.